test/runAllTests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+REPO_PATH="/path/to/emender-tests"
```

Comment:
Can you please add a comment that the user needs to change the value before running the script? Something like:

    REPO_PATH="/path/to/emender-tests"  # IMPORTANT: Replace with an absolute path to emender-tests